auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -72,7 +72,7 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
     @JvmOverloads
     public constructor(
         auth0: Auth0,
-        networkingClient: NetworkingClient = DefaultClient(auth0.connectTimeoutInSeconds)
+        networkingClient: NetworkingClient = DefaultClient()
```

Comment:
When I make updates to the DefaultClient constructor params, I'll wire the timeout/logging, etc params into the DefaultClient constructor here.